docs/releasing.md

```diff
@@ -59,6 +53,13 @@ git push origin uportal-app-framework-maven-<version>
 
 + Release maven artifacts from [central staging repository][central Maven staging repository]
 
+
+### Node release
+
++ Update the npm package by running `npm version <update_type>` where <update_type> is based on the selection above: patch, minor, or major. This will: change the version in `package.json`, commit that (locally), and make a tag in the format `vx.y.z`. Read more about that [on npm docs](https://docs.npmjs.com/getting-started/publishing-npm-packages#updating-the-package).
```

Comment:
:speaking_head: link text `on npm docs` leaves a person using a screen reader thinking "what on the npm docs?" 
For better link accessibility

``` markdown
[Read more about publishing on npm docs](https://docs.npmjs.com/getting-started/publishing-npm-packages#updating-the-package).
```

or

``` markdown
Read more about publishing on [npm docs](https://docs.npmjs.com/getting-started/publishing-npm-packages#updating-the-package).
```